team_member_test.go

```diff
@@ -9,6 +9,8 @@ import (
 )
 
 func TestTeamMembersList(t *testing.T) {
+	skipIfFreeOnly(t)
```

Comment:
Hmmm, so technically free orgs _are_ able to add/remove/list team members and create team tokens, it's just that they only have access to one team (the owners team) and can't create new ones - that is, this test fails because it tries to create a new team, not because the team members list API endpoint is actually unavailable to free orgs!

Eventually it might be nice to have tests that exercise the owners team too, but in the meantime, do you think it's worth adding a note about the owners team thing to explain why these tests are `skipIfFreeOnly`?

Reply:
Yup, will do!